### PR TITLE
Migrate Test Users creation URL to CI Gateway

### PR DESCRIPTION
**Pull request by taymoork2**
Hotfix, no JIRA

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

File: .env.default [hunk 1]
```diff
@@ -1,17 +1,14 @@
...
-WEBEX_SCOPE="Identity:OAuthClient webexsquare:get_conversation webexsquare:admin Identity:SCIM spark:people_read spark:rooms_read spark:rooms_write spark:memberships_read spark:memberships_write spark:messages_read spark:messages_write spark:applications_read spark:applications_write spark:teams_read spark:teams_write spark:team_memberships_read spark:team_memberships_write spark:bots_read spark:bots_write spark:kms"
-SCOPE="Identity:OAuthClient webexsquare:get_conversation webexsquare:admin Identity:SCIM spark:people_read spark:rooms_read spark:rooms_write spark:memberships_read spark:memberships_write spark:messages_read spark:messages_write spark:applications_read spark:applications_write spark:teams_read spark:teams_write spark:team_memberships_read spark:team_memberships_write spark:bots_read spark:bots_write spark:kms"
```
Comment (taymoork2) [inline .env.default:7]: I had to get rid of these and the tests started running

**Comment by adamweeks** [comment]
Since we have other teams still using this package, let's add fallback support for the `WEBEX_TEST_USERS_CONVERSATION_SERVICE_URL` env var.

File: packages/node_modules/@webex/test-users/src/index.js [hunk 2]
```diff
@@ -156,7 +156,7 @@ export function createTestUser(options = {}) {
   const clientId = options.clientId || process.env.WEBEX_CLIENT_ID;
   const clientSecret = options.clientSecret || process.env.WEBEX_CLIENT_SECRET;
   const idbrokerUrl = options.idbrokerUrl || process.env.IDBROKER_BASE_URL;
-  const conversationServiceUrl = options.conversationServiceUrl || process.env.WEBEX_TEST_USERS_CONVERSATION_SERVICE_URL;
+  const cigServiceUrl = options.cigServiceUrl || process.env.WEBEX_TEST_USERS_CI_GATEWAY_SERVICE_URL;
```
Comment (taymoork2) [inline packages/node_modules/@webex/test-users/src/index.js:159]: @adamweeks that means that `WEBEX_TEST_USERS_CONVERSATION_SERVICE_URL` would need to be added back at the end here, correct?

File: packages/node_modules/@webex/test-users/README.md [hunk 3]
```diff
@@ -38,7 +38,7 @@ The test users library falls back to a few environment variables if they aren't
 
 - `WEBEX_CLIENT_ID` - The Webex client ID that has access to create test users
 - `WEBEX_CLIENT_SECRET` - The Webex client secret for the given client id
-- `WEBEX_TEST_USERS_CONVERSATION_SERVICE_URL` - The Webex url for conversation service to create test users
```
Comment (adamweeks) [inline packages/node_modules/@webex/test-users/README.md:41]: Could you add a deprecation warning to the readme as well regarding this change?

**Review by adamweeks — APPROVED** [review]
Thanks for getting this out so quickly!

**Pull request merged**